stringFormat/index.js

```diff
@@ -0,0 +1 @@
+export const numberFormat = (number) => number.toLocaleString();
```

Comment:
You don't need parentheses around the variable 'number' in the code.

Reply:
Thank you for pointing that out @hirwaaldo1  but if you use Prettier as a code formatting tool, it will automatically format the code, including the placement of brackets.